src/store/rbac/selectors.ts

```diff
@@ -20,7 +20,7 @@ export const isCostModelWritePermission = (state: RootState) => {
   if (app === 'cost-management' && resource === '*' && operation === '*') {
     return true;
   }
-  if (resource === 'rate' && operation === 'write') {
+  if ((resource === 'rate' || resource === 'cost_model') && operation === 'write') {
```

Comment:
Are you keeping `rate` for backward compatibility, aren't you?

Reply:
@boaz0 Yes.